src/primitives/polyline/styled.rs

```diff
@@ -10,14 +10,50 @@ use crate::{
         Primitive, Rectangle,
     },
     style::{PrimitiveStyle, Styled},
+    transform::Transform,
 };
 
+impl<'a, C> Styled<Polyline<'a>, PrimitiveStyle<C>>
+where
+    C: PixelColor,
+{
+    /// Compute the bounding box of the non-translated polyline.
+    pub(in crate::primitives::polyline) fn original_bounding_box(&self) -> Rectangle {
```

Comment:
This could use a more descriptive function name IMO:
```suggestion
    pub(in crate::primitives::polyline) fn untranslated_bounding_box(&self) -> Rectangle {
```